packages/bits/demo/src/components/demo/table/table-docs/table-docs.example.component.html

```diff
@@ -291,6 +291,9 @@ <h3>Virtual Scroll with Sticky Header</h3>
     <code>tableStickyHeader</code> directive supports runtime state switch by just binding a boolean value to it <code>[tableStickyHeader]="false"</code>
     or taking the <code>TableStickyHeaderDirective</code> reference via <code>ViewChild</code> decorator and updating the <code>tableStickyHeader</code> property.
 </nui-message>
+<nui-message [allowDismiss]="false" type="info">
+    To have columns with fixed width set the <code>columnWidth</code> property with the desired width.
```

Comment:
For even better clarity maybe this?

```suggestion
    To have columns with fixed width set the <code>columnWidth</code> input on the <code>nuiColumnDef</code> directive to the desired width in pixels.
```